js/particle/particle_animation.js

```diff
@@ -77,6 +83,7 @@ export class ParticleAnimation {
     this.absorptionTextGroup.remove();
   }
 
+  // NOTE maybe just one timout would suffice
```

Comment:
pisałem - notka to coś do zastanowienia się; KTOKOLWIEK się za to weźmie będzie dobrze, ale nie jest to wysoki priorytet